### DISCUSSION TITLE
mask mode file

### DISCUSSION THREAD

**Discussion by DH-HAN** [open-mmlab/mmediting#1224] [How-to]
Are there any example for inpainting mask mode 'file' ??
I can't find detail infomration of mask mode from file

My detail Situation is like this

I have a photo with people, and the thing that i'd like to do is delete people and only leave background

in this situation how can i set my train data pipeline

For example I have three file

mask_of_people.png(input_1)
image_of_with_people.png(input_2)
image_of_none_people.png(Output and ground Truth)

Input_1
![I-211026_O01008_W06_F0006](https://user-images.githubusercontent.com/58062066/187157325-e3d1389c-fe33-45a8-9537-c3230e7cbf8d.jpg)

Input_2
![I-211026_O01008_W06_F0007_Mask](https://user-images.githubusercontent.com/58062066/187157447-84c4b560-ef48-4862-a1bf-561113274e0c.jpg)

Grount_Truth and the thing that i want to get as my output
![Ground_Truth_1](https://user-images.githubusercontent.com/58062066/187157479-cec47df7-1ab3-4fce-9df4-1cf35a620246.jpg)

**Comment by zengyh1900** [comment]
Hi @DH-HAN ,
MMEditing has provided a user-friendly demo for a quick run. 
You can check the example here: https://mmediting.readthedocs.io/en/1.x/user_guides/3_inference.html#run-a-demo-of-image-inpainting 
You can check the source code here: https://github.com/open-mmlab/mmediting/blob/1.x/demo/inpainting_demo.py
